.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	root: true,
+	extends: ['@hokify'],
+	parserOptions: {
+		project: './tsconfig.eslint.json'
```

Comment:
`eslint` can be dropped.

Reply:
eslint was intended, I'm not saying it has to stay, but in general my opinion is that eslint and prettier are a very well known standard nowadays :)